### PR TITLE
// fix generator combination language

### PR DESCRIPTION
**Pull request by kelu95**

### CODE REVIEW

File: src/Adapter/Attribute/AttributeDataProvider.php [hunk 1]
```diff
@@ -84,6 +84,8 @@ public static function getAttributeIdsByGroup($id_group, $not_null = false)
      */
     public function getProductCombinations($idProduct)
     {
+        $locales = $this->container->get('prestashop.adapter.legacy.context')->getLanguages();
```
Comment (Shudrum) [inline src/Adapter/Attribute/AttributeDataProvider.php]: Why not `$languages`? Maybe more explicit for a `getLanguages()` return?

Comment (kelu95) [inline src/Adapter/Attribute/AttributeDataProvider.php]: its named likethis everywhere, so what did you suggest ?

Comment (Shudrum) [inline src/Adapter/Attribute/AttributeDataProvider.php]: Yep I see… I don't like the old "everyone do that", but its not the more important modification to do right now.

Comment (kpodemski) [inline src/Adapter/Attribute/AttributeDataProvider.php]: well, also it alwas was `$id_product` not `$idProduct`, I see many changes in naming convention, not sure if for better...

Comment (Shudrum) [inline src/Adapter/Attribute/AttributeDataProvider.php]: Naming convention is discussed, but on the Adapter, it is not really problem.

**Comment by kelu95** [comment]
thanks

**Pull request merged**